src/data/buffer/basic.lean

```diff
@@ -28,6 +28,24 @@ lemma ext : ∀ {b₁ b₂ : buffer α}, to_list b₁ = to_list b₂ → b₁ =
   rw eq_of_heq (h.trans a₂.to_list_to_array)
 end
 
+lemma ext_iff {b₁ b₂ : buffer α} : to_list b₁ = to_list b₂ ↔ b₁ = b₂ :=
+⟨ext, λ h, h ▸ rfl⟩
```

Comment:
An `ext_iff` lemma usually has the `ext` lemma as the implication from right to left.
```suggestion
lemma ext_iff {b₁ b₂ : buffer α} : b₁ = b₂ ↔ to_list b₁ = to_list b₂ :=
⟨λ h, h ▸ rfl, ext⟩
```